examples/flask/dash-in-flask/README.md

```diff
@@ -0,0 +1,89 @@
+# Integrating Dash apps into a Flask app
+
+Interactive Flask application, with Dash applications integrated using:
+- Direct hosting: A simple approach more fitting for smaller projects is passing the Flask instance directly into each Dash app. However, since each of these app is a component of the main Flask app, adding too many Dash app components may overload the Flask server. See `host/app.py` for implementation. For more information, see the [documentation](https://flask.palletsprojects.com/en/2.3.x/appcontext/).
```

Comment:
```suggestion
- Direct hosting: A simple approach more fitting for smaller projects is passing the Flask instance directly into each Dash app. However, since each of these app is a component of the main Flask app, adding too many Dash app components may overload the Flask server. See `host/app.py` for implementation. For more information, see the [Flask documentation](https://flask.palletsprojects.com/en/2.3.x/appcontext/).
```

Reply:
I noticed that the reference was wrong, so I [updated](https://github.com/ploomber/doc/pull/263/commits/ff8a135793f7d68499d2ee3446742f65c223702a#diff-650a7c63682e837f4a1145a5daec3593a5b15ea17e717de26ff9970781e37563R4) it as well